lib/Data/Validate/Sanctions.pm

```diff
@@ -87,6 +103,7 @@ sub set_sanction_file {    ## no critic (RequireArgUnpacking)
 }
 
 sub get_sanction_file {
+    $sanction_file //= _default_sanction_file();
```

Comment:
Will be better to use `state` here ?

Reply:
in some cases, the state declaration does not apply because we are using the internal scope of the variable in the `set_sanction_file` and the `get_sanction_file` subroutines.